Formula/cppad.rb

```diff
@@ -1,9 +1,10 @@
 class Cppad < Formula
   desc "Differentiation of C++ Algorithms"
   homepage "https://www.coin-or.org/CppAD"
-  url "https://www.coin-or.org/download/source/CppAD/cppad-20171128.epl.tgz"
-  version "20171128"
-  sha256 "36f9a4b6c16d720c20547c0a896c0b89fd5f05641dba34274257a1700f85d13b"
+  # Stable versions have numbers of the form 201x0000.y
```

Comment:
We should add an audit rule for this.

Reply:
Although I guess if we're just going off of GitHub it won't come up.

Reply:
Yeah, that's the reason I moved to GitHub, so we avoid the issue altogether